src/styles/overrides.scss

```diff
@@ -104,28 +115,28 @@
     }
 }
 
-.pc-navigation {
-    z-index:  101;
-
-    .desktop.dc-root_wide-format & .container-fluid {
-        padding: 30px;
-
-        & .col {
-            padding: 0;
-        }
-    }
-
-    .mobile.dc-root_wide-format & .container-fluid {
-        padding: 12px;
-
-        & .col {
-            padding: 0;
-        }
-    }
+.pc-mobile-navigation {
+    z-index: 119;
 }
 
 .gc-simple-consent.gc-analytics {
     right: 0;
     left: unset;
     z-index: 101;
 }
+
+// md
+
+@media (max-width: map-get($screenBreakpoints, 'md') - 1) {
+    .dc-toc {
+        height: fit-content !important; /* stylelint-disable-line declaration-no-important */
```

Comment:
better not to use `!important`

Reply:
height: fit-content !important is needed to overlap styles from the style attribute in .dc-toc

Reply:
check if it is possible to move this redefinition directly to the toc styles